### DISCUSSION TITLE
Pemenang AWS POP Quiz 03 - Periode 05 Juni 2023 - 09 Juni 2023

### DISCUSSION THREAD

**Discussion by rioastamal** [awsugid/forum#12] [AWS Pop Quiz]
## Pemenang AWS Quiz 03

Selamat kepada **Sebastianus Sembara** yang menjadi pemenang AWS POP Quiz 03. AWS Credit senilai **$25** akan dikirim oleh panitia ke alamat email pemenang.

## Jawaban dari Quiz 03 (Tema: AWS Global Infrastructure)

1. Global Infrastructure yang didesain oleh Amazon Web Services (AWS) memiliki banyak manfaat, salah satunya adalah memastikan sumber daya dan layanan selalu beroperasi dari waktu ke waktu sehingga kita dapat terus mengaksesnya. Hal tersebut juga dikenal dengan istilah...

- [ ] Scalability
- [x] Availability
- [ ] Security
- [ ] Flexibility

2. Kode region untuk Asia Pacific (Jakarta) adalah...

- [ ] ap-southeast-1
- [ ] ap-southeast-2
- [x] ap-southeast-3
- [ ] ap-southeast-4

3. Manakah diantara dua pernyataan berikut yang benar tentang konsep Region dan Availability Zone (AZ) di AWS?

- [ ] Dalam sebuah Availability Zone (AZ) terdapat tiga atau lebih region
- [x] Dalam sebuah region terdapat tiga atau lebih Availability Zone (AZ)

## Tanya jawab

Jika ada pertanyaan seputar soal-soal diatas bisa langsung posting di thread ini. Pembahasan jawaban yang lebih lengkap dapat dilihat di channel YouTube [AWS User Group Indonesia](https://www.youtube.com/watch?v=irL97jT8a84) atau [Indonesia Belajar](https://www.youtube.com/watch?v=OZNA1DoA5MY)

**Comment by baralogi** [comment]
Thank you 🚀
